src/main/java/com/iexec/core/replicate/ReplicateStatusUpdateError.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2021 IEXEC BLOCKCHAIN TECH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.iexec.core.replicate;
+
+public enum ReplicateStatusUpdateError {
+    UNKNOWN_REPLICATE("Cannot update replicate, could not get replicate [chainTaskId:{}, UpdateRequest:{}]"),
```

Comment:
The String interpolation here will only work when using `log.info()` which makes them strongly coupled to each other. We can either remove it from here or use something like `getMessage(chainTaskId, updateRequest (...))` with generic string interpolation (`[chainTaskId:%s, UpdateRequest:%s, ....]`).

Reply:
You're right. I'll see what fits best. Thanks!